demos-europe/demosplan-core
# F t35367 adjust authored date export value format

Ticket:
https://yaits.demos-deutschland.de/T35367

Description:
The exported xls file contained a unix timestamp instead of a readable
formatted date string. This change formats this value so it is the same as the other date value
present within the export (d.m.Y)

### How to review/test
get an up to date regio db, check out branch and download the xls statement(s) as described within the ticket.
The date value should look like `01.11.2023` instead of the unix timeStamp - lokal exampleExport:
[Abwägungstabelle-10-11-2023-14 07.xlsx](https://github.com/demos-europe/demosplan-core/files/13320402/Abwagungstabelle-10-11-2023-14.07.xlsx)
